eng/config/PublishData.json

```diff
@@ -121,120 +121,57 @@
       "version": "2.10.*",
       "nuget": [ ],
       "vsix": null,
-      "channels": [ "dev15.9" ],
+      "channels": [ ],
       "vsBranch": "rel/d15.9",
       "vsMajorVersion": 15
     },
-    "dev16.0-vs-deps": {
```

Comment:
Should we also link to the servicing doc like we did here:

https://github.com/dotnet/roslyn-tools/blob/9528c5b1c7c625f87b55ec9e805750e4b9b3d7b6/src/GitHubCreateMergePRs/config.xml#L3

Reply:
sicne this isn't jsonc that is a little more complicated, but maybe I can add a "comment" property somewhere.

Reply:
Perhaps a comment at the point where the PublishData is parsed in the bulid script?

Reply:
Or is it parsed by an external service? I don't remember.

Reply:
Added a comment in the publishdata.json.